src/App.jsx

```diff
@@ -14,33 +15,79 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState([...goodsFromServer]);
+  const [reversed, setReversed] = useState(false);
+  const [sortMethod, setSortMethod] = useState(''); // 'alphabet' | 'length' | ''
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const sortAlphabetically = () => {
+    setSortMethod('alphabet');
+    setVisibleGoods([...goodsFromServer].sort((a, b) => a.localeCompare(b)));
+    setReversed(false);
```

Comment:
- Why do you set the reverse state in the sort handler?
- `alphabet`, `length` are magic values, create constants
- Create a function that accepts goods array, sort type and reversed state and depending on these parameters prepare them. And move this function outside the function